tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 3.7
-envlist = clean,{py36,py37,py38,py39,py310}-{linux,darwin,windows},report
+envlist = clean,{py37,py38,py39,py310,py311}-{linux,darwin,windows},report
```

Comment:
Aha - I completely missed updating tox.ini for 3.11... Thanks for catching this!

Reply:
On second thought this is probably not needed anymore, let me try (on another branch) to remove this and see what happens...

Reply:
Okay, now `tox.ini` no longer contains hardcoded Python version numbers. The only downside is, if you have multiple Python installed and want to run a test across all of them by simply executing `tox`, it wouldn't work anymore.

Reply:
I actually do make use of the envlist to run "tox -p" locally, so I think I'd like to keep that. Is there any way to leave that in but avoid having to repeat things down in the "depends"? If not, I'm ok with just taking the COVERAGE_FILE changes.

On another note, there's a new dependency on "liboqs" (for post-Quantum encryption) that we might want to try and add here. On macOS, it's available via HomeBrew, so that should be pretty easy. Perhaps that'd be good enough for now to make codecov happy. Unfortunately, I don't see a precompiled version of liboqs in Ubuntu, so it would probably need to be compiled from source for Ubuntu and Windows. More info is at https://github.com/open-quantum-safe/liboqs.

Reply:
Noted, I'll restore the envlist and try to reduce some repetition if possible. I'll also look into building and installing liboqs.

Reply:
Thanks! I made the change to add liboqs for macOS already, and it seemed to work nicely.